### PR TITLE
Leaderboard (Page structure)

### PR DESCRIPTION
**Pull request by edubew**
# Milestone 1
> In this project milestone, I did the following:
- Install webpack and configure linters
- Create the page outline of the leaderboard app with Html
- Create a form dynamically with JavaScript
- Create the scores table dynamically
- Add styling

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
Hi @edubew , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

**You have submitted a project, without a descriptive README file.** 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: package.json [hunk 1]
```diff
@@ -21,12 +21,21 @@
   "devDependencies": {
     "@babel/core": "^7.18.6",
```
Comment (ShoiraTa) [inline package.json:23]: - [ ] I cannot start application by running 'npm start', this command should open the document on the browser and update it whenever you make changes, so you don't have to run `npm run build` and refresh page every time. All you need to do is updating package.json file and add `start` script: 
```json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @edubew,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

_Check the comments under the review._ 

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack.config.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ module.exports = {
   devtool: 'source-map',
   devServer: {
     static: {
-      directory: path.resolve(__dirname, 'dist')
+      directory: path.resolve(__dirname, 'dist'),
     },
     port: 3000,
     open: true,
```
Comment (brytebee) [inline webpack.config.js:21]: - [ ] **Required:** As stated by the previous reviewers, I am unable to run your app, please ensure that `npm start` runs and opens your app in a browser.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @edubew👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by uwadonat — APPROVED** [review]
Hi @edubew, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# Highlights
- Descriptive pull request message and Readme file ✔️
- Clean and organized code ✔️
appr3

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**